src/main/java/no/nav/foreldrepenger/oversikt/innhenting/journalføringshendelse/HentDataFraJoarkForHåndteringTask.java

```diff
@@ -77,5 +87,23 @@ public void doTask(ProsessTaskData p) {
             LOG.info("Journalføringshendelse av dokumenttypen {} på sak {} ignoreres", dokumentType, saksnummer);
         }
     }
+
+    private void sendMinSideBeskjed(EnkelJournalpost journalpost) {
+        var saksnummer = journalpost.saksnummer();
+        var fødselsnummer = journalpost.fødselsnummerAvsenderMottaker().value();
+        var dokumentTypeHoveddokument = journalpost.hovedtype();
+        if (dokumentTypeHoveddokument.erFørstegangssøknad() || dokumentTypeHoveddokument.erEndringssøknad()) {
+            var ytelseType = dokumentTypeHoveddokument.gjelderYtelse();
+            var erEndringssøknad = dokumentTypeHoveddokument.erEndringssøknad();
+            var task = ProsessTaskData.forProsessTask(BrukernotifikasjonBeskjedVedMottattSøknadTask.class);
+            task.setSaksnummer(saksnummer);
+            task.setCallIdFraEksisterende();
+            task.setProperty(YTELSE_TYPE, ytelseType.name());
+            task.setProperty(ER_ENDRINGSSØKNAD, Boolean.toString(erEndringssøknad));
+            task.setProperty(FØDSELSNUMMER, fødselsnummer);
+            task.setProperty(SAKSNUMMER, saksnummer);
```

Comment:
Jeg mangler properties for sekvens osv - usikker på hva som gjelder. Denne er uavhengig av annen behandling, så kan kjøres fritt.

Reply:
blir dobbelt opp med setSaksnummer og setProperty(SAKSNUMMER

Reply:
Riktig at man ikke trenger sekvens og gruppe her ja